.gitignore

```diff
@@ -1,32 +1,28 @@
-# Compiled nonsense that does not belong in *source* control
-/build
-/bin
-/.gradle
-/out
-/classes
-
-# Run stuff
-/minecraft
-/run
-/logs
-
-# IDE nonsense that could go in source control but really shouldn't
+# Private
+
+.gradle
+private.gradle
+
+
+# IDE stuff
+
+.idea
 .classpath
 .project
-.metadata
 .settings
-*.launch
-*.iml
-.idea
-*.ipr
-*.iws
 
-# Sekrit files
-private.properties
-private.gradle
+# Logs
+
+logs
+
+# Build bits
+
+run
+bin
+build
+out
+classes
+
+# Naughty filesystems
 
-# Files from bad operating systems :^)
-Thumbs.db
-.DS_Store
-local.properties
 .directory
```

Comment:
Will this really deal with Thumbs.db, .DS_Store, and local.properties? I don't think it's really necessary to change the gitignore; it's been working fine so far.

Reply:
Dunno, that’s an automatic thing. I don’t thing it worths deleting

Reply:
I've run into issues in the past where these, if not gitignore'd, will appear in source when you don't want them to. I don't think we should have any gitignore changes.

Reply:
Ah, right, they’re deleted, not added

Reply:
yeah, definitely don't remove these, we have a lot of different systems creating a lot of hidden files

but local.properties should move down into the gradle private files section